txscript/data/script_tests.json

```diff
@@ -551,6 +556,370 @@
+["DATA_40 0x7a{40}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_40 to ALTSTACK"],
+["DATA_41 0x7a{41}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_41 to ALTSTACK"],
+["DATA_42 0x7a{42}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_42 to ALTSTACK"],
+["DATA_43 0x7a{43}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_43 to ALTSTACK"],
+["DATA_44 0x7a{44}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_44 to ALTSTACK"],
+["DATA_45 0x7a{45}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_45 to ALTSTACK"],
+["DATA_46 0x7a{46}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_46 to ALTSTACK"],
+["DATA_47 0x7a{47}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_47 to ALTSTACK"],
+["DATA_48 0x7a{48}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_48 to ALTSTACK"],
+["DATA_49 0x7a{49}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_49 to ALTSTACK"],
+["DATA_50 0x7a{50}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_50 to ALTSTACK"],
+["DATA_51 0x7a{51}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_51 to ALTSTACK"],
+["DATA_52 0x7a{52}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_52 to ALTSTACK"],
+["DATA_53 0x7a{53}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_53 to ALTSTACK"],
+["DATA_54 0x7a{54}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_54 to ALTSTACK"],
+["DATA_55 0x7a{55}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_55 to ALTSTACK"],
+["DATA_56 0x7a{56}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_56 to ALTSTACK"],
+["DATA_57 0x7a{57}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_57 to ALTSTACK"],
+["DATA_58 0x7a{58}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_58 to ALTSTACK"],
+["DATA_59 0x7a{59}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_59 to ALTSTACK"],
+["DATA_60 0x7a{60}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_60 to ALTSTACK"],
+["DATA_61 0x7a{61}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_61 to ALTSTACK"],
+["DATA_62 0x7a{62}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_62 to ALTSTACK"],
+["DATA_63 0x7a{63}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_63 to ALTSTACK"],
+["DATA_64 0x7a{64}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_64 to ALTSTACK"],
+["DATA_65 0x7a{65}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_65 to ALTSTACK"],
+["DATA_66 0x7a{66}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_66 to ALTSTACK"],
+["DATA_67 0x7a{67}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_67 to ALTSTACK"],
+["DATA_68 0x7a{68}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_68 to ALTSTACK"],
+["DATA_69 0x7a{69}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_69 to ALTSTACK"],
+["DATA_70 0x7a{70}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_70 to ALTSTACK"],
+["DATA_71 0x7a{71}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_71 to ALTSTACK"],
+["DATA_72 0x7a{72}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_72 to ALTSTACK"],
+["DATA_73 0x7a{73}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_73 to ALTSTACK"],
+["DATA_74 0x7a{74}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_74 to ALTSTACK"],
+["DATA_75 0x7a{75}", "TOALTSTACK 1", "NONE", "OK", "Push DATA_75 to ALTSTACK"],
+["NOP", "TOALTSTACK 1", "NONE", "ERR_INVALID_STACK_OPERATION"],
```

Comment:
There should be a test similar to this, but which is in the same script since the alt stack is cleared between scripts.